.github/workflows/jacoco_report.yml

```diff
@@ -3,12 +3,13 @@ name: Measure coverage
 on:
   pull_request:
     branches:
-      - duy/main
+      - master
```

Comment:
Would this always checkout master branch?
Is this what we want to run a report on?

Reply:
This is triggered when someone is creating a PR into master.

Reply:
I see, this is for PR targeting master.
I'm not the biggest expert on GitHub Actions, but I'd expect to see a `ref` to the PR branch in the checkout operation.

Can you confirm that what we check out is the PR branch?